src/main/java/org/apache/maven/plugin/compiler/AbstractCompilerMojo.java

```diff
@@ -531,12 +541,12 @@
 
     /**
      * File extensions to check timestamp for incremental build.
-     * Default contains only <code>class</code> and <code>jar</code>.
+     * Default contains only {@code class} and {@code jar}.
      *
      * @since 3.1
      */
-    @Parameter
-    private List<String> fileExtensions;
+    @Parameter( defaultValue = "class,jar" )
```

Comment:
When you add `defaultValue` the information about it will added by plugin tools, so we can remove manual description.